index.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8">
+    <title>Flickr-Poc</title>
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<body>
+<div class="container">
+
+    <h1>Flickr-Puf</h1>
+
+    <h6>Gallery</h6>
+    <div class="button">
+        <button id="prev">
+            < <span class="offscreen">Previous</span>
```

Comment:
Please use the HTML entity for the less/greater than signs. I believe they are: `&lt;` & `&gt;`